src/rpm_ostree/cli_status.rs

```diff
@@ -18,6 +18,8 @@ pub struct DeploymentJSON {
     booted: bool,
     #[serde(rename = "base-checksum")]
     base_checksum: Option<String>,
+    #[serde(rename = "coreos-assembler.basearch")]
```

Comment:
Check out the JSON at https://github.com/coreos/zincati/issues/54#issuecomment-508087569. This is actually nested under `base-commit-meta`.